docs/source/dataframe-parquet.rst

```diff
@@ -0,0 +1,234 @@
+Dask Dataframe and Parquet
+==========================
+
+.. currentmodule:: dask.dataframe
+
+Parquet_ is a popular, columnar file format designed for efficient data storage
+and retrieval. Dask dataframe includes `read_parquet` and `to_parquet`
+functions/methods for reading and writing parquet files respectively. Here we
+document these methods, and provide some tips and best practices.
+
+Reading Parquet Files
+---------------------
+
+.. autosummary::
+    read_parquet
+
+Dask dataframe provides a `read_parquet` function for reading parquet files and
+datasets. Its first argument is one of:
+
+- A path to a single parquet file
+- A path to a directory of parquet files (files with ``.parquet`` or ``.parq`` extension)
+- A `glob string`_ expanding to one or more parquet file paths
+- A list of parquet file paths
+
+These paths can be local, or point to some remote filesystem (e.g. S3, HDFS,
+...) by prepending the path with a protocol.
+
+.. code-block:: python
+
+    >>> import dask.dataframe as dd
+
+    # Load a single local parquet file
+    >>> df = dd.read_parquet("path/to/mydata.parquet")
+
+    # Load a directory of local parquet files
+    >>> df = dd.read_parquet("path/to/my/parquet/")
+
+    # Load a directory of parquet files from S3
+    >>> df = dd.read_parquet("s3://bucket-name/my/parquet/")
+
+Note that for remote filesystems you many need to configure credentials. When
+possible we recommend handling these external to Dask through
+filesystem-specific configuration files/environment variables (e.g. the
+``~/.aws/config`` file for S3). Alternatively, you can pass configuration on to
+the fsspec_ backend through the ``storage_options`` kwarg:
```

Comment:
Does it make sense to link the "Connect to remote data" page somewhere here (https://github.com/dask/dask/blob/main/docs/source/how-to/connect-to-remote-data.rst)?

Maybe at the end of this section ("To read more about connecting to remote data, see ...")?

Reply:
Thanks, I knew we had a page like that but I couldn't find the link anywhere.